cli/azd/internal/repository/infra_confirm.go

```diff
@@ -274,6 +274,14 @@ func PromptPort(
 	svc appdetect.Project) (int, error) {
 	if svc.Docker == nil || svc.Docker.Path == "" { // using default builder from azd
 		if svc.Language == appdetect.Java {
+			for _, dep := range svc.Dependencies {
+				switch dep {
+				case appdetect.JavaEurekaServer:
+					return 8761, nil
+				case appdetect.JavaConfigServer:
+					return 8888, nil
```

Comment:
Is it possible that one project has both the 2 dependencies?

Reply:
It's not common practices from microservice perspective, and also not recommended in the community. But I think it's possible if we still want it.